arfna-backend/src/main/java/org/arfna/util/security/SecurityUtility.java

```diff
@@ -0,0 +1,127 @@
+    private static final String LOCAL_HIBERNATE_CREDENTIALS = "hibernate_creds.json";
+
+    private static SecurityKey SECURITY_KEY;
+    private static Map HIBERNATE_PROPERTIES_MAP;
+    private static SecretsManagerClient SECRET_CLIENT;
+
+
+    public static SecurityKey getSecurityKeys() {
+        if (SECURITY_KEY == null) {
+            loadSecurityKey();
+        }
+        return SECURITY_KEY;
+    }
+
+    public static Map getHibernatePropertiesCredentials() {
+        if (HIBERNATE_PROPERTIES_MAP == null) {
+            loadHibernatePropertiesMap();
+        }
+        return HIBERNATE_PROPERTIES_MAP;
+    }
+
+    private static void loadHibernatePropertiesMap() {
+        HIBERNATE_PROPERTIES_MAP = EnvironmentHelper.isRuntimeProd() ?
+                loadHibernateProperties(getSecretString(HIBERNATE_PROPS_SECRET)) :
+                loadHibernateProperties(getHibernateProps());
+    }
+
+    private static void loadSecurityKey() {
+        SECURITY_KEY = EnvironmentHelper.isRuntimeProd() ?
+                getSecurityKeyFromSecret() : getSecurityKeyFromDev();
+    }
+
+    private static SecurityKey getSecurityKeyFromDev() {
+        return GsonHelper.getGsonWithPrettyPrint().fromJson(getSecurityKeyJsonDev(), SecurityKey.class);
+    }
+
+    private static Map loadHibernateProperties(String hibernateJson) {
+        return GsonHelper.getGson().fromJson(hibernateJson, Map.class);
+    }
+
+    private static SecurityKey getSecurityKeyFromSecret() {
+        String secret = getSecretString(CREDENTIALS_SECRET_NAME);
+        assert secret != null;
+        JsonObject root = JsonParser.parseString(secret).getAsJsonObject();
+        return SecurityKey
+            .builder()
+            .withAwsCredentials(
+                    root.getAsJsonPrimitive("aws_role.access_key").getAsString(),
+                    root.getAsJsonPrimitive("aws_role.secret_key").getAsString())
+            .withMailerSendCredentials(
+                    root.getAsJsonPrimitive("mailersend.api_token").getAsString()
+            );
+    }
+
+    private static String getSecretString(String key) {
+        try {
+            GetSecretValueRequest valueRequest = GetSecretValueRequest.builder()
+                    .secretId(key)
+                    .build();
+            GetSecretValueResponse valueResponse = getSecretClient().getSecretValue(valueRequest);
+            return valueResponse.secretString();
+        } catch (SecretsManagerException e) {
+            ArfnaLogger.exception(SecurityUtility.class, "Unable to fetch secret " + key + ": " + e.awsErrorDetails().errorMessage(), e);
+        }
+        return null;
+    }
+
+    private static SecretsManagerClient getSecretClient() {
+        if (SECRET_CLIENT == null) {
+            Region region = Region.US_WEST_2;
+            SECRET_CLIENT = SecretsManagerClient.builder()
+                    .region(region)
+                    .build();
+        }
+        return SECRET_CLIENT;
+    }
```

Comment:
@MajoretteKay This is where we load from secrets, only in Linux environment. The EC2 instance where you are running the code has access to the secret. For local, here we still read from a file.